static/client.js

```diff
@@ -1,4 +1,24 @@
 (function () {
+  // TODO: make this a template variable?
+  const replace_urls = {
+    "api.blaseball.com": "",
+    "blaseball-configs.s3.us-west-2.amazonaws.com": "/s3-bucket"
```

Comment:
also need to replace blaseball-configs-qa due to some early prod builds having the wrong bucket